src/documentation/text/01_Preface/task-definition.tex

```diff
@@ -0,0 +1,6 @@
+Die vorliegende Seminararbeit thematisiert den Anwendungsfall eines Ticketverkaufs für Veranstaltungen.
+Kunden und Verwalter erhalten über spezielle Clients Zugriff auf eine Verwaltungskomponente, dessen Implementierung auf einem Aktorensystem basiert.
+Hierbei muss die Implementierung des Systems alle Anwendungsfälle erfüllen.
+Dafür müssen zunächst die Anforderungen an das System gesammelt werden, bevor eine Unterteilung in funktionale und nicht-funktionale Anforderungen stattfindet.
```

Comment:
Dafür müssen zunächst die Anforderungen an das System gesammelt werden bevor aus diesen Anwendungsfälle generiert werden können? Irgendwie sowas?

Reply:
Klingt gut. Ist aber **eigentlich** fachlich "falsch", Da man eig. aus den UseCases die Anforderungen generiert und nicht aus den Anforderungen die UseCases. Haben das in der Arbeit anders strukturiert, da der Funk die Anforderungen bereits in der Aufgabenstellung sehr konkret definiert hat.
Vielleicht:
```suggestion
Dafür müssen zunächst die Softwareanforderungen konkretisiert werden, die die Entwicklung und Realisierung begleiten und die notwendigen Rahmenbedingungen für eine erfolgreiche Konzeption schaffen.
```

Reply:
Ich würde den Satz rauswerfen und den Satz in Zeile 3 durch `Hierbei muss die Implementierung des Systems vordefinierte Anforderungen erfüllen.` ersetzen. Das passt m.E. sowieso besser.
Die Einleitung können wir am Ende noch verlängern, falls erforderlich.
Was meinst du dazu?

Reply:
@lucascheurer Dein Kommentar wurde mir noch nicht angezeigt. Eventuell können wir unsere Vorschläge kombinieren.